tools/nxfuse/src/filemtd.c

```diff
@@ -161,17 +161,25 @@ static ssize_t filemtd_write(FAR struct file_dev_s *priv, size_t offset, FAR con
 	uint8_t srcvalue;
 	uint8_t newvalue;
 	size_t seekpos;
-
+	size_t bufsize = sizeof(buf);
```

Comment:
Can we use a macro for this?

Reply:
sizeof is not function it is kind of operator, so I think there's no benefit from macro.

Reply:
I mean use MACRO instead of 128
`char buf[128]`

Reply:
yes, using constant value is always not good. I will update.